salt-master/README.md

```diff
@@ -0,0 +1,46 @@
+Build salt-master image
+=======================
+
+```
+docker build -t moonshot/salt-master metalk8s/salt-master
```

Comment:
this does not run.. if you want multiple names go for `docker build -t name1:tag name2:tag`
and you should add the directory of Dockerfile to the command

Reply:
That's not multiple name, but image name (`-t moonshot/salt-master`) and source directory (`metalk8/salt-master`) where to look for dockerfile